content/docs/1.8.0/best-practices.md

```diff
@@ -99,6 +103,10 @@ We recommend running your Kubernetes cluster on one of the following versions. T
 
 Referenced to https://endoflife.date/kubernetes.
 
+### CoreDNS Setup
+
+Ensure that CoreDNS runs with at least 2 replicas to maintain high availability. This setup minimizes interruptions in the DNS resolution if one CoreDNS pod experiences a temporary disruption.
```

Comment:
Can we improve the preflight command to check this? It doesn't need to be a criterion, but it can serve as a warning to inform users about their cluster DNS lacking high availability.

P.S. It is uncertain whether users will choose a different DNS solution instead of CoreDNS. However, we don't need to focus on this.

Reply:
https://github.com/longhorn/cli/pull/120